openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -116,6 +118,7 @@ def _create_jwt(
         'filters': filters or [],
         'is_restricted': is_restricted,
         'email_verified': user.is_active,
+        'roles': create_role_auth_claim_for_user(user),
```

Comment:
i'm wondering if it would be good to conserve space by only adding the roles array if the claim is not empty?

Reply:
Good question. I'm not sure what the best practices are in this case, as in, should other systems expect a jwt with the same keys available all the time?

Reply:
Okay, works for me. I can make this change